packages/api/orm/entity/Environment.ts

```diff
@@ -0,0 +1,22 @@
+import { Application } from "./Application";
+import { Column, Entity, ManyToOne } from "typeorm";
+import { ProvisionableEntity } from "./ProvisionableEntity";
+
+@Entity()
+export class Environment extends ProvisionableEntity {
+  @Column({ type: String, nullable: false, length: 100 })
+  name: string;
+
+  @ManyToOne(() => Application, (application) => application.environments)
+  application: Application;
+
+  // operators - simple-array or AppEnvOperator[] ?
+  @Column({ type: "simple-array", default: "" })
+  administrators: string[];
+
+  @Column({ type: "simple-array", default: "" })
+  contributors: string[];
+
+  @Column({ type: "simple-array", default: "" })
+  readOnlyOperators: string[];
```

Comment:
Can these move to the `ProvisionableEntity` class? It seems like `Portfolio`, `Environment`, and `Application` all have them. Am I missing another specializing class?

Reply:
Yes, I think that is a good improvement.  Those three entities are the totality.

Reply:
Moved in 54e87a13ffaeead8556cbb1c46f680dc2bcb2bd6